src/include/configuration/configuration.h

```diff
@@ -2,17 +2,33 @@
 //
 //                         Peloton
 //
-// config.h
+// configuration.h
 //
-// Identification: src/include/common/config.h
+// Identification: src/include/configuration/configuration.h
 //
-// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+// Copyright (c) 2015-2017, Carnegie Mellon University Database Group
 //
 //===----------------------------------------------------------------------===//
-
 #pragma once
 
-#include <gflags/gflags.h>
+// internal use only in
+// configuration/configuration_manager.cpp
+#ifdef __PELOTON_CONFIG__
```

Comment:
Yes, it is useless now. At the beginning, I tried to unfold the content of this header file in a function defined in configuration_manager.cpp. But gflags doesn't allow me to do so.